config/sources/families/bcm2711.conf

```diff
@@ -7,16 +7,27 @@ export KERNELDIR='linux-rpi'                # Avoid sharing a source tree with o
 export FK__PUBLISHED_KERNEL_VERSION="raspi" # flash kernel (FK) configuration
 export FK__KERNEL_PACKAGES=""
 export RASPI_ROOT_FS_LABEL="armbian"
+export CPUMIN=500000
+export CPUMAX=2000000
+export GOVERNOR=ondemand
 
 case "${BRANCH}" in
 
-current)
+legacy)
 	export RASPI_DISTRO_KERNEL=yes # This will cause board to include distro's prebuilt kernel, not from source
 	;;
 
+current)
+        export RASPI_DISTRO_KERNEL=no
+	export KERNELSOURCE='https://github.com/raspberrypi/linux'
+        export KERNELBRANCH="branch:rpi-5.10.y"
+        export KERNELPATCHDIR="${LINUXFAMILY}-${BRANCH}"
+        export LINUXCONFIG="linux-${LINUXFAMILY}-${BRANCH}"
+	;;
 edge)
 	export RASPI_DISTRO_KERNEL=no
-	export KERNELBRANCH="branch:linux-5.15.y"
+        export KERNELSOURCE='https://github.com/raspberrypi/linux'
+        export KERNELBRANCH="branch:rpi-5.15.y"
```

Comment:
should this be 5.16?

Reply:
5.16.y comes here after some testings of 5.15.y ... since in this case 5.10.y is going out.

Reply:
Or we simply start "current" with 5.16.y?

Reply:
Sounds good as well. See also: https://forums.raspberrypi.com/viewtopic.php?t=322879